### ISSUE TITLE
JPA service management web app fails to init underlying DB (PostgreSQL)

### ISSUE THREAD

**Issue by dima767** [apereo/cas#2378]
CAS: `5.0.3-SNAPSHOT`
DB: PostgreSQL: `9.6` 

For easier reference, on MacOS, I use super simple point-and-click Postgres app: https://postgresapp.com/

Vanilla `cas-management` app set up to init initial schema and data with the following:

```groovy
compile "org.apereo.cas:cas-server-support-jpa-service-registry:${project.'cas.version'}"
```

```bash
#JPA Service Registry
cas.serviceRegistry.initFromJson=true
cas.serviceRegistry.jpa.driverClass=org.postgresql.Driver
cas.serviceRegistry.jpa.url=jdbc:postgresql://localhost:5432/cas_services
cas.serviceRegistry.jpa.user=myuser
cas.serviceRegistry.jpa.dialect=org.hibernate.dialect.PostgreSQL94Dialect
cas.serviceRegistry.jpa.healthQuery=SELECT 1
cas.serviceRegistry.jpa.ddlAuto=create-drop
```

On deploy to external Tomcat fails with this:

```java
Hibernate: alter table RegisteredServiceImpl_Props drop constraint FK1xan7uamsa94y2451jgksjkj4
Hibernate: alter table RegisteredServiceImpl_Props drop constraint FK5ghaknoplphay7reury7n3vcm
Hibernate: drop table if exists RegexRegisteredService cascade
Hibernate: drop table if exists RegexRegisteredServiceProperty cascade
Hibernate: drop table if exists RegisteredServiceImpl_Props cascade
Hibernate: drop sequence if exists hibernate_sequence
Hibernate: create sequence hibernate_sequence start 1 increment 1
Hibernate: create table RegexRegisteredService (expression_type VARCHAR(15) DEFAULT 'ant' not null, id int8 not null, access_strategy oid, attribute_release oid, description varchar(255) not null, evaluation_order int4 not null, logo varchar(255), logout_type int4, logout_url varchar(255), mfa_policy oid, name varchar(255) not null, proxy_policy oid, public_key oid, required_handlers oid, serviceId varchar(255) not null, theme varchar(255), username_attr oid, bypassApprovalPrompt boolean, clientId varchar(255), clientSecret varchar(255), generateRefreshToken boolean, jsonFormat boolean, jwks varchar(2147483647), signIdToken boolean, encryptAssertions boolean, metadataCriteriaDirection varchar(2147483647), metadataCriteriaPattern varchar(2147483647), metadataCriteriaRemoveEmptyEntitiesDescriptors boolean, metadataCriteriaRemoveRolelessEntityDescriptors boolean, metadataCriteriaRoles varchar(2147483647), metadataLocation varchar(255), metadataMaxValidity int8, metadataSignatureLocation varchar(2147483647), requiredAuthenticationContextClass varchar(2147483647), requiredNameIdFormat varchar(2147483647), signAssertions boolean, signResponses boolean, primary key (id))
Hibernate: create table RegexRegisteredServiceProperty (id int8 not null, property_values oid, primary key (id))
Hibernate: create table RegisteredServiceImpl_Props (AbstractRegisteredService_id int8 not null, properties_id int8 not null, properties_KEY varchar(255) not null, primary key (AbstractRegisteredService_id, properties_KEY))
Hibernate: alter table RegisteredServiceImpl_Props add constraint UK_i2mjaqjwxpvurc6aefjkx5x97 unique (properties_id)
Hibernate: alter table RegisteredServiceImpl_Props add constraint FK1xan7uamsa94y2451jgksjkj4 foreign key (properties_id) references RegexRegisteredServiceProperty
Hibernate: alter table RegisteredServiceImpl_Props add constraint FK5ghaknoplphay7reury7n3vcm foreign key (AbstractRegisteredService_id) references RegexRegisteredService
Hibernate: select abstractre0_.id as id2_0_, abstractre0_.access_strategy as access_s3_0_, abstractre0_.attribute_release as attribut4_0_, abstractre0_.description as descript5_0_, abstractre0_.evaluation_order as evaluati6_0_, abstractre0_.logo as logo7_0_, abstractre0_.logout_type as logout_t8_0_, abstractre0_.logout_url as logout_u9_0_, abstractre0_.mfa_policy as mfa_pol10_0_, abstractre0_.name as name11_0_, abstractre0_.proxy_policy as proxy_p12_0_, abstractre0_.public_key as public_13_0_, abstractre0_.required_handlers as require14_0_, abstractre0_.serviceId as service15_0_, abstractre0_.theme as theme16_0_, abstractre0_.username_attr as usernam17_0_, abstractre0_.bypassApprovalPrompt as bypassA18_0_, abstractre0_.clientId as clientI19_0_, abstractre0_.clientSecret as clientS20_0_, abstractre0_.generateRefreshToken as generat21_0_, abstractre0_.jsonFormat as jsonFor22_0_, abstractre0_.jwks as jwks23_0_, abstractre0_.signIdToken as signIdT24_0_, abstractre0_.encryptAssertions as encrypt25_0_, abstractre0_.metadataCriteriaDirection as metadat26_0_, abstractre0_.metadataCriteriaPattern as metadat27_0_, abstractre0_.metadataCriteriaRemoveEmptyEntitiesDescriptors as metadat28_0_, abstractre0_.metadataCriteriaRemoveRolelessEntityDescriptors as metadat29_0_, abstractre0_.metadataCriteriaRoles as metadat30_0_, abstractre0_.metadataLocation as metadat31_0_, abstractre0_.metadataMaxValidity as metadat32_0_, abstractre0_.metadataSignatureLocation as metadat33_0_, abstractre0_.requiredAuthenticationContextClass as require34_0_, abstractre0_.requiredNameIdFormat as require35_0_, abstractre0_.signAssertions as signAss36_0_, abstractre0_.signResponses as signRes37_0_, abstractre0_.expression_type as expressi1_0_ from RegexRegisteredService abstractre0_
2017-02-07 10:30:52,546 ERROR [org.hibernate.engine.jdbc.spi.SqlExceptionHelper] - <ERROR: relation "regexregisteredservice" does not exist
  Position: 1727>
2017-02-07 10:30:52,551 WARN [org.springframework.boot.context.embedded.AnnotationConfigEmbeddedWebApplicationContext] - <Exception encountered during context initialization - cancelling refresh attempt: org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'manageRegisteredServicesMultiActionController' defined in class path resource [org/apereo/cas/mgmt/config/CasManagementWebAppConfiguration.class]: Unsatisfied dependency expressed through method 'manageRegisteredServicesMultiActionController' parameter 0; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'servicesManager': Invocation of init method failed; nested exception is javax.persistence.PersistenceException: org.hibernate.exception.SQLGrammarException: could not extract ResultSet>
Hibernate: alter table RegisteredServiceImpl_Props drop constraint FK1xan7uamsa94y2451jgksjkj4
Hibernate: alter table RegisteredServiceImpl_Props drop constraint FK5ghaknoplphay7reury7n3vcm
2017-02-07 10:30:52,554 ERROR [org.hibernate.tool.schema.internal.SchemaDropperImpl$DelayedDropActionImpl] - <HHH000478: Unsuccessful: alter table RegisteredServiceImpl_Props drop constraint FK5ghaknoplphay7reury7n3vcm>
Hibernate: drop table if exists RegexRegisteredService cascade
Hibernate: drop table if exists RegexRegisteredServiceProperty cascade
Hibernate: drop table if exists RegisteredServiceImpl_Props cascade
Hibernate: drop sequence if exists hibernate_sequence
2017-02-07 10:30:52,584 ERROR [org.springframework.boot.SpringApplication] - <Application startup failed>
org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'manageRegisteredServicesMultiActionController' defined in class path resource [org/apereo/cas/mgmt/config/CasManagementWebAppConfiguration.class]: Unsatisfied dependency expressed through method 'manageRegisteredServicesMultiActionController' parameter 0; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'servicesManager': Invocation of init method failed; nested exception is javax.persistence.PersistenceException: org.hibernate.exception.SQLGrammarException: could not extract ResultSet
	at org.springframework.beans.factory.support.ConstructorResolver.createArgumentArray(ConstructorResolver.java:749) ~[spring-beans-4.3.4.RELEASE.jar:4.3.4.RELEASE]
	at org.springframework.beans.factory.support.ConstructorResolver.instantiateUsingFactoryMethod(ConstructorResolver.java:467) ~[spring-beans-4.3.4.RELEASE.jar:4.3.4.RELEASE]
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.instantiateUsingFactoryMethod(AbstractAutowireCapableBeanFactory.java:1128) ~[spring-beans-4.3.4.RELEASE.jar:4.3.4.RELEASE]
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBeanInstance(AbstractAutowireCapableBeanFactory.java:1022) ~[spring-beans-4.3.4.RELEASE.jar:4.3.4.RELEASE]
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:512) ~[spring-beans-4.3.4.RELEASE.jar:4.3.4.RELEASE]
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:482) ~[spring-beans-4.3.4.RELEASE.jar:4.3.4.RELEASE]
	at org.springframework.beans.factory.support.AbstractBeanFactory$1.getObject(AbstractBeanFactory.java:306) ~[spring-beans-4.3.4.RELEASE.jar:4.3.4.RELEASE]
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:230) ~[spring-beans-4.3.4.RELEASE.jar:4.3.4.RELEASE]
	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:302) ~[spring-beans-4.3.4.RELEASE.jar:4.3.4.RELEASE]
	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:197) ~[spring-beans-4.3.4.RELEASE.jar:4.3.4.RELEASE]
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.preInstantiateSingletons(DefaultListableBeanFactory.java:754) ~[spring-beans-4.3.4.RELEASE.jar:4.3.4.RELEASE]
	at org.springframework.context.support.AbstractApplicationContext.finishBeanFactoryInitialization(AbstractApplicationContext.java:866) ~[spring-context-4.3.4.RELEASE.jar:4.3.4.RELEASE]
	at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:542) ~[spring-context-4.3.4.RELEASE.jar:4.3.4.RELEASE]
	at org.springframework.boot.context.embedded.EmbeddedWebApplicationContext.refresh(EmbeddedWebApplicationContext.java:122) ~[spring-boot-1.4.1.RELEASE.jar:1.4.1.RELEASE]
	at org.springframework.boot.SpringApplication.refresh(SpringApplication.java:761) [spring-boot-1.4.1.RELEASE.jar:1.4.1.RELEASE]
	at org.springframework.boot.SpringApplication.refreshContext(SpringApplication.java:371) [spring-boot-1.4.1.RELEASE.jar:1.4.1.RELEASE]
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:315) [spring-boot-1.4.1.RELEASE.jar:1.4.1.RELEASE]
	at org.springframework.boot.web.support.SpringBootServletInitializer.run(SpringBootServletInitializer.java:151) [spring-boot-1.4.1.RELEASE.jar:1.4.1.RELEASE]
	at org.springframework.boot.web.support.SpringBootServletInitializer.createRootApplicationContext(SpringBootServletInitializer.java:131) [spring-boot-1.4.1.RELEASE.jar:1.4.1.RELEASE]
	at org.springframework.boot.web.support.SpringBootServletInitializer.onStartup(SpringBootServletInitializer.java:86) [spring-boot-1.4.1.RELEASE.jar:1.4.1.RELEASE]
	at org.springframework.web.SpringServletContainerInitializer.onStartup(SpringServletContainerInitializer.java:169) [spring-web-4.3.4.RELEASE.jar:4.3.4.RELEASE]
	at org.apache.catalina.core.StandardContext.startInternal(StandardContext.java:5244) [catalina.jar:8.0.32]
	at org.apache.catalina.util.LifecycleBase.start(LifecycleBase.java:147) [catalina.jar:8.0.32]
	at org.apache.catalina.core.ContainerBase.addChildInternal(ContainerBase.java:725) [catalina.jar:8.0.32]
	at org.apache.catalina.core.ContainerBase.addChild(ContainerBase.java:701) [catalina.jar:8.0.32]
	at org.apache.catalina.core.StandardHost.addChild(StandardHost.java:717) [catalina.jar:8.0.32]
	at org.apache.catalina.startup.HostConfig.deployDescriptor(HostConfig.java:585) [catalina.jar:8.0.32]
	at org.apache.catalina.startup.HostConfig$DeployDescriptor.run(HostConfig.java:1794) [catalina.jar:8.0.32]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) [?:1.8.0_92]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [?:1.8.0_92]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) [?:1.8.0_92]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) [?:1.8.0_92]
	at java.lang.Thread.run(Thread.java:745) [?:1.8.0_92]
Caused by: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'servicesManager': Invocation of init method failed; nested exception is javax.persistence.PersistenceException: org.hibernate.exception.SQLGrammarException: could not extract ResultSet
	at org.springframework.beans.factory.annotation.InitDestroyAnnotationBeanPostProcessor.postProcessBeforeInitialization(InitDestroyAnnotationBeanPostProcessor.java:137) ~[spring-beans-4.3.4.RELEASE.jar:4.3.4.RELEASE]
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.applyBeanPostProcessorsBeforeInitialization(AbstractAutowireCapableBeanFactory.java:408) ~[spring-beans-4.3.4.RELEASE.jar:4.3.4.RELEASE]
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.initializeBean(AbstractAutowireCapableBeanFactory.java:1575) ~[spring-beans-4.3.4.RELEASE.jar:4.3.4.RELEASE]
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:553) ~[spring-beans-4.3.4.RELEASE.jar:4.3.4.RELEASE]
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:482) ~[spring-beans-4.3.4.RELEASE.jar:4.3.4.RELEASE]
	at org.springframework.beans.factory.support.AbstractBeanFactory$1.getObject(AbstractBeanFactory.java:306) ~[spring-beans-4.3.4.RELEASE.jar:4.3.4.RELEASE]
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:230) ~[spring-beans-4.3.4.RELEASE.jar:4.3.4.RELEASE]
	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:302) ~[spring-beans-4.3.4.RELEASE.jar:4.3.4.RELEASE]
	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:202) ~[spring-beans-4.3.4.RELEASE.jar:4.3.4.RELEASE]
	at org.springframework.beans.factory.config.DependencyDescriptor.resolveCandidate(DependencyDescriptor.java:207) ~[spring-beans-4.3.4.RELEASE.jar:4.3.4.RELEASE]
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.doResolveDependency(DefaultListableBeanFactory.java:1131) ~[spring-beans-4.3.4.RELEASE.jar:4.3.4.RELEASE]
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.resolveDependency(DefaultListableBeanFactory.java:1059) ~[spring-beans-4.3.4.RELEASE.jar:4.3.4.RELEASE]
	at org.springframework.beans.factory.support.ConstructorResolver.resolveAutowiredArgument(ConstructorResolver.java:835) ~[spring-beans-4.3.4.RELEASE.jar:4.3.4.RELEASE]
	at org.springframework.beans.factory.support.ConstructorResolver.createArgumentArray(ConstructorResolver.java:741) ~[spring-beans-4.3.4.RELEASE.jar:4.3.4.RELEASE]
	... 32 more
Caused by: javax.persistence.PersistenceException: org.hibernate.exception.SQLGrammarException: could not extract ResultSet
	at org.hibernate.internal.ExceptionConverterImpl.convert(ExceptionConverterImpl.java:147) ~[hibernate-core-5.2.2.Final.jar:5.2.2.Final]
	at org.hibernate.internal.ExceptionConverterImpl.convert(ExceptionConverterImpl.java:155) ~[hibernate-core-5.2.2.Final.jar:5.2.2.Final]
	at org.hibernate.query.internal.AbstractProducedQuery.list(AbstractProducedQuery.java:1407) ~[hibernate-core-5.2.2.Final.jar:5.2.2.Final]
	at org.hibernate.Query.getResultList(Query.java:417) ~[hibernate-core-5.2.2.Final.jar:5.2.2.Final]
	at org.apereo.cas.services.JpaServiceRegistryDaoImpl.load(JpaServiceRegistryDaoImpl.java:40) ~[cas-server-support-jpa-service-registry-5.0.3-SNAPSHOT.jar:5.0.3-SNAPSHOT]
	at org.apereo.cas.services.JpaServiceRegistryDaoImpl$$FastClassBySpringCGLIB$$fcd53dc2.invoke(<generated>) ~[cas-server-support-jpa-service-registry-5.0.3-SNAPSHOT.jar:5.0.3-SNAPSHOT]
	at org.springframework.cglib.proxy.MethodProxy.invoke(MethodProxy.java:204) ~[spring-core-4.3.4.RELEASE.jar:4.3.4.RELEASE]
	at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.invokeJoinpoint(CglibAopProxy.java:720) ~[spring-aop-4.3.4.RELEASE.jar:4.3.4.RELEASE]
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:157) ~[spring-aop-4.3.4.RELEASE.jar:4.3.4.RELEASE]
	at org.springframework.transaction.interceptor.TransactionInterceptor$1.proceedWithInvocation(TransactionInterceptor.java:99) ~[spring-tx-4.3.4.RELEASE.jar:4.3.4.RELEASE]
	at org.springframework.transaction.interceptor.TransactionAspectSupport.invokeWithinTransaction(TransactionAspectSupport.java:282) ~[spring-tx-4.3.4.RELEASE.jar:4.3.4.RELEASE]
	at org.springframework.transaction.interceptor.TransactionInterceptor.invoke(TransactionInterceptor.java:96) ~[spring-tx-4.3.4.RELEASE.jar:4.3.4.RELEASE]
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:179) ~[spring-aop-4.3.4.RELEASE.jar:4.3.4.RELEASE]
	at org.springframework.aop.framework.CglibAopProxy$DynamicAdvisedInterceptor.intercept(CglibAopProxy.java:655) ~[spring-aop-4.3.4.RELEASE.jar:4.3.4.RELEASE]
	at org.apereo.cas.services.JpaServiceRegistryDaoImpl$$EnhancerBySpringCGLIB$$d342d03f.load(<generated>) ~[cas-server-support-jpa-service-registry-5.0.3-SNAPSHOT.jar:5.0.3-SNAPSHOT]
	at org.apereo.cas.services.DefaultServicesManagerImpl.load(DefaultServicesManagerImpl.java:146) ~[cas-server-core-services-5.0.3-SNAPSHOT.jar:5.0.3-SNAPSHOT]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_92]
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:1.8.0_92]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_92]
	at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_92]
	at org.springframework.beans.factory.annotation.InitDestroyAnnotationBeanPostProcessor$LifecycleElement.invoke(InitDestroyAnnotationBeanPostProcessor.java:366) ~[spring-beans-4.3.4.RELEASE.jar:4.3.4.RELEASE]
	at org.springframework.beans.factory.annotation.InitDestroyAnnotationBeanPostProcessor$LifecycleMetadata.invokeInitMethods(InitDestroyAnnotationBeanPostProcessor.java:311) ~[spring-beans-4.3.4.RELEASE.jar:4.3.4.RELEASE]
	at org.springframework.beans.factory.annotation.InitDestroyAnnotationBeanPostProcessor.postProcessBeforeInitialization(InitDestroyAnnotationBeanPostProcessor.java:134) ~[spring-beans-4.3.4.RELEASE.jar:4.3.4.RELEASE]
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.applyBeanPostProcessorsBeforeInitialization(AbstractAutowireCapableBeanFactory.java:408) ~[spring-beans-4.3.4.RELEASE.jar:4.3.4.RELEASE]
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.initializeBean(AbstractAutowireCapableBeanFactory.java:1575) ~[spring-beans-4.3.4.RELEASE.jar:4.3.4.RELEASE]
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:553) ~[spring-beans-4.3.4.RELEASE.jar:4.3.4.RELEASE]
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:482) ~[spring-beans-4.3.4.RELEASE.jar:4.3.4.RELEASE]

...

Caused by: org.postgresql.util.PSQLException: ERROR: relation "regexregisteredservice" does not exist
  Position: 1727
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2458) ~[postgresql-9.4.1211.jre7.jar:9.4.1211.jre7]
	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:2158) ~[postgresql-9.4.1211.jre7.jar:9.4.1211.jre7]
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:291) ~[postgresql-9.4.1211.jre7.jar:9.4.1211.jre7]
	at org.postgresql.jdbc.PgStatement.executeInternal(PgStatement.java:432) ~[postgresql-9.4.1211.jre7.jar:9.4.1211.jre7]
...
```

and then after trying to deploy cas app first with the jpa services registry, it seems to init the DB just fine with tables and initial data, but then again, dropping `cas-management` app into the mix, fails miserably with: 

```java
2017-02-07 11:04:58,569 INFO [org.apereo.cas.mgmt.web.CasManagementWebApplicationServletInitializer] - <The following profiles are active: native>
Hibernate: select abstractre0_.id as id2_0_, abstractre0_.access_strategy as access_s3_0_, abstractre0_.attribute_release as attribut4_0_, abstractre0_.description as descript5_0_, abstractre0_.evaluation_order as evaluati6_0_, abstractre0_.logo as logo7_0_, abstractre0_.logout_type as logout_t8_0_, abstractre0_.logout_url as logout_u9_0_, abstractre0_.mfa_policy as mfa_pol10_0_, abstractre0_.name as name11_0_, abstractre0_.proxy_policy as proxy_p12_0_, abstractre0_.public_key as public_13_0_, abstractre0_.required_handlers as require14_0_, abstractre0_.serviceId as service15_0_, abstractre0_.theme as theme16_0_, abstractre0_.username_attr as usernam17_0_, abstractre0_.bypassApprovalPrompt as bypassA18_0_, abstractre0_.clientId as clientI19_0_, abstractre0_.clientSecret as clientS20_0_, abstractre0_.generateRefreshToken as generat21_0_, abstractre0_.jsonFormat as jsonFor22_0_, abstractre0_.jwks as jwks23_0_, abstractre0_.signIdToken as signIdT24_0_, abstractre0_.encryptAssertions as encrypt25_0_, abstractre0_.metadataCriteriaDirection as metadat26_0_, abstractre0_.metadataCriteriaPattern as metadat27_0_, abstractre0_.metadataCriteriaRemoveEmptyEntitiesDescriptors as metadat28_0_, abstractre0_.metadataCriteriaRemoveRolelessEntityDescriptors as metadat29_0_, abstractre0_.metadataCriteriaRoles as metadat30_0_, abstractre0_.metadataLocation as metadat31_0_, abstractre0_.metadataMaxValidity as metadat32_0_, abstractre0_.metadataSignatureLocation as metadat33_0_, abstractre0_.requiredAuthenticationContextClass as require34_0_, abstractre0_.requiredNameIdFormat as require35_0_, abstractre0_.signAssertions as signAss36_0_, abstractre0_.signResponses as signRes37_0_, abstractre0_.expression_type as expressi1_0_ from RegexRegisteredService abstractre0_
2017-02-07 11:05:04,421 ERROR [org.hibernate.engine.jdbc.spi.SqlExceptionHelper] - <ERROR: relation "regexregisteredservice" does not exist
  Position: 1727>
2017-02-07 11:05:04,427 WARN [org.springframework.boot.context.embedded.AnnotationConfigEmbeddedWebApplicationContext] - <Exception encountered during context initialization - cancelling refresh attempt: org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'manageRegisteredServicesMultiActionController' defined in class path resource [org/apereo/cas/mgmt/config/CasManagementWebAppConfiguration.class]: Unsatisfied dependency expressed through method 'manageRegisteredServicesMultiActionController' parameter 0; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'servicesManager': Invocation of init method failed; nested exception is javax.persistence.PersistenceException: org.hibernate.exception.SQLGrammarException: could not extract ResultSet>
2017-02-07 11:05:04,458 ERROR [org.springframework.boot.SpringApplication] - <Application startup failed>
org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'manageRegisteredServicesMultiActionController' defined in class path resource [org/apereo/cas/mgmt/config/CasManagementWebAppConfiguration.class]: Unsatisfied dependency expressed through method 'manageRegisteredServicesMultiActionController' parameter 0; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'servicesManager': Invocation of init method failed; nested exception is javax.persistence.PersistenceException: org.hibernate.exception.SQLGrammarException: could not extract ResultSet

...
```

**Comment by Condor70** [comment]
Isn't this the same issue as #2250?

Unfortunately, that fix was for the 5.1 branch and was not backported to 5.0.

**Comment by mmoayyed** [comment]
Closed via https://github.com/apereo/cas/commit/df333388b2308578b519c943f051ec795decea66
